cores/arduino/RingBuffer.h

```diff
@@ -27,7 +27,9 @@
 // using a ring buffer (I think), in which head is the index of the location
 // to which to write the next incoming character and tail is the index of the
 // location from which to read.
-#define SERIAL_BUFFER_SIZE 64
+#ifndef defined(SERIAL_BUFFER_SIZE)//allows to define the user the buffer size
+#define SERIAL_BUFFER_SIZE 128//Double the buffer size to get larger streams like from GPS
```

Comment:
I'd be up for bumping this to 256 by default.

The comment about doubling the buffer size should be removed before this pull request can be merged.